### ISSUE TITLE
Documentation still refering (indirectly) to create-solid

### ISSUE THREAD

**Issue by vigenere23** [solidjs/solid-start#282]
The documentation specifies using the following command to quick-start :

```shell
npx init solid
```

However, I've found 2 problems with this command :

1. `init solid` refers to `create-solid` as of now (instead of `solid-start`). Should the documentation specify to use `solid@next` until it reaches official version `0.1.0`?
2. Using `npx` results in an error from npm : `could not determine executable to run`. Is there any previous steps that needs to be done? Using `npm` instead of `npx` does fix the problem.

In conclusion, I think the `npx init solid` should be changed to `npm init solid@next`.

**Comment by ryansolid** [comment]
I published it on latest. `create-solid` is in this repo now. So it should be `npm init solid`. We should be past using `@next`. I just changed this last night. 

However, I have found on some machines that have previously installed the old version won't automatically download and run the new version. That I want to figure out if anyone else has had the same thing happen to them.

**Comment by ryansolid** [comment]
I think this is all as it should be now. No more references to npx. We moved on to the 0.x.x versions so I think we are all good. Thanks for reporting.
